src/openrct2/ride/SteamPosition.hpp

```diff
@@ -39,7 +39,7 @@ namespace OpenRCT2::Math::Trigonometry
      * Where Y1 represents the angle of pitch in degrees
      * Using physical pitch calculated by gravity
      */
-    constexpr CoordsXY pitchToDirectionVector[60] = {
+    constexpr CoordsXY pitchOffsetsFromPhysics[60] = {
```

Comment:
This name means a lot less than before. Previously it meant converting a pitch to a direction vector now it just means garbage and physics has nothing to do with this. Can you also apply the coding style as mentioned in the last comment.